### PR TITLE
Wrap project name in project list and change default visible columns

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
I was wrong to remove project name wrapping in the first place. I surrender 🏳️
- https://github.com/cityofaustin/atd-data-tech/issues/11500

## Testing
**URL to test:**  [Netlify](https://deploy-preview-981--atd-moped-main.netlify.app/moped/session/signin)

**Steps to test:**
1. Create a project with a very long name. See name wrap in project list.
2. To test the column config, use your debug console to delete the `mopedColumnConfig` from local storage.
3. Confirm the default visibility matches [here](https://github.com/cityofaustin/atd-data-tech/issues/11500).

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Review by frankhereford — APPROVED** [review]
Works great - looks perfect on the listing page. Code peeked - this is ship-shape. 🚀

This may warrant a spin off issue, but feels out of scope of this one. Verrrry long names might benefit from some reasonable truncation on the summary page perhaps with the actual name hidden in a tooltip like you did with the dates in the activity log recently? This may be a solution without a problem - not sure.

image

File: moped-editor/src/views/projects/signalProjectTable/RenderSignalLink.js [hunk 1]
```diff
@@ -24,7 +13,6 @@ const RenderSignalLink = ({ signals }) => {
               rel="noopener noreferrer"
             >
               {signal.signal_id}
-              <OpenInNewIcon className={classes.openInNewIcon} />
```
Comment (chiaberry) [inline moped-editor/src/views/projects/signalProjectTable/RenderSignalLink.js:27]: 🧹

File: moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js [hunk 2]
```diff
@@ -57,6 +57,34 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+/**
+ * Default column display (if no config in local storage)
+ */
+const DEFAULT_HIDDEN_COLS = {
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js:63]: 😬  at past me

**Review by chiaberry — APPROVED** [review]

**Review by mddilley — APPROVED** [review]
The wrap is back! Looks good, default columns work good - all good. 👍 🙏 😎 I agree with Frank's note about reallllyyy long descriptions which probably don't happen that often. Worth a spin off issue for sure, because I can't think of a great solution beyond Frank's suggestion for truncation that doesn't involve scroll or a growing title bar area.

**Comment by johnclary** [comment]
thanks y'all! these are some [first class project names](https://moped.austinmobility.io/moped/projects/1838) - lol.

agreed—i will make a long overdue backlog issue to deal with project names. i think some combo of truncating the next with a "reveal" button, plus maybe actually limiting the character count of project names? we definitely need to standardize our conventions a bit.

**Pull request merged**